web/src/beta/pages/Page.tsx

```diff
@@ -55,16 +55,24 @@ const PageWrapper: React.FC<Props> = ({ sceneId, projectId, workspaceId, childre
   ) : (
     <>
       <GlobalModal />
-      {childrenWithProps}
+      {renderItem({
+        sceneId,
+        projectId: currentProjectId,
+        workspaceId: currentWorkspaceId,
+        stories: scene?.stories ?? [],
```

Comment:
The only tab that cares about stories is the storytelling page, but here you are treating it like all pages care and need them. 
Can you setup a useStorytellingQuery inside of the API, and fetch the stories, etc at that level? 
Anyways, it'll use cached values since scene has already been queried, but it saves us from long props lists with things that are only related to specific tabs, pages, components, etc. and keeps intent obvious and simpler 🙏🏻

Reply:
We talked that it is good to consider in another PR since the idea needs experiment/investigation.
So mark as resolved :bow: